src/_gettsim/transfers/rente.py

```diff
@@ -339,11 +346,7 @@ def age_of_retirement(
     """
     if rentner:
         out = (
-            jahr_renteneintr
-            + (monat_renteneintr - 1) / 12
-            - geburtsjahr
-            + (geburtsmonat - 1) / 12
-            - (1 / 12)
+            jahr_renteneintr - geburtsjahr + (monat_renteneintr - geburtsmonat - 1) / 12
```

Comment:
Can you please make a note somewhere that we should decide to either use year - month in separate variables throughout (like here) or make single variables out of it (like `birthdate_decimal`) ? There are arguments for both; I think there are few arguments for mixing. But that should not stop this PR.

Reply:
I opened issue #641 for this